lambda/handlers/intentHandler.js

```
@@ -115,108 +115,127 @@ const NextPrayerTimeIntentHandler = {
+          now,
+          prayerNameFromData
+        );
+      }
+      switch (parseInt(prayerNameResolvedId)) {
+        case 5:
+          // Extract only the Jumu'ah times
+          const jumuaTimes = [
+            mosqueTimesData.jumua,
+            mosqueTimesData.jumua2,
+            mosqueTimesData.jumua3,
+          ];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)